infrastructure/serve/network.tf

```diff
@@ -0,0 +1,31 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+resource "azurerm_subnet" "web_app" {
```

Comment:
```suggestion
resource "azurerm_subnet" "serve" {
```

Reply:
given this is a delegated subnet for web apps I'm not sure it makes sense to call it `serve`(?) 
or is it best to have a serve subnet and a webapp one?

Reply:
Yeah added this comment before I saw the delegation I think xD perhaps we say `serve_webapps`?